official/vision/serving/export_saved_model_lib.py

```diff
@@ -35,8 +35,8 @@ def export_inference_graph(
     batch_size: Optional[int],
     input_image_size: List[int],
     params: cfg.ExperimentConfig,
-    checkpoint_path: str | None,
     export_dir: str,
+    checkpoint_path: Optional[str] = None,
```

Comment:
Can you keep the original order of arguments by moving `checkpoint_path` before `export_dir`?

Reply:
Updated the suggested changes.